src/assignee-list/AssigneeList.js

```diff
@@ -9,11 +9,14 @@ class AssigneeList extends Component {
   }
 
   componentDidMount() {
-    this.setState({ assignees: ["Self", "Annie", "Kyle", "Drew" ]});
+    var that = this;
```

Comment:
We can get rid of the this/that line actually. I just updated and will push with the next commit.